src/app/_components/NavBar/NetworkLabel.tsx

```diff
@@ -77,22 +88,22 @@ export const NetworkLabel: FC<{ network: Network }> = ({ network }) => {
             </Badge>
           ) : (
             <Badge
-              color={'purple.600'}
-              bg={'purple.100'}
+              color={colorMode === 'light' ? 'purple.600' : 'pruple.300'}
```

Comment:
We usually use `useColorModeValue` on these instances.

Reply:
done